### ISSUE TITLE
Changed notes would benefit from parenthetical "(Modifed)"

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag2ict#699]
Almost all notes which are not new either (1) use WCAG 2.2 note verbatim or (2) apply word substitution as described.

A very few notes are rewritten.  They are substantively the same, just re-written for the context of WCAG2ICT.

In these case, I suggest that instead of "Note 1" such notes should be written as "Note 1 (modified)" similar to how we have things like "Note 3 (Added)".

I raised this issue on the 19 June call, and PR #632 is an example.

**Comment by maryjom** [comment]
@bruce-usab Thanks for opening this. Assigning to @daniel-montalvo and myself. Daniel will have to do a change to the classes and document build script to allow us to have a "(Modified)" parenthetic available for use. Then I can add the class to those instances where we replaced the note, specifically to SC 4.1.2, and potentially also to the "accessibility supported" definition where we replaced all 5 notes with a single note (currently marked as "Added").

**Comment by daniel-montalvo** [comment]
Thanks @bruce-usab @maryjom 

I will be putting together a PR for this. IT still would require a little bit of editorial work to identify which notes we want to mark as "modified" and then add the corresponding class, we can decide as we see the PR in-place.

**Comment by daniel-montalvo** [comment]
Hi @bruce-usab @maryjom 

See [this pr preview](https://deploy-preview-701--wcag2ict.netlify.app/#applying-sc-4-1-2-name-role-value-to-non-web-documents-and-software)

I've added the logic to treat these "modiified" notes as such.

@bruce-usab let me know if that's what you want and then we'll merge the script updates and probably open another issue for the editorial work to determine which notes will be marked as "modified".

**Comment by bruce-usab** [comment]
A bit off topic, but related to #682 is that there are a few more _notes_ which use the word "must":
- Note 2 of 4.1.1 Parsing
- Note 1 of accessibility supported in glossary
- Note 2 of web page in glossary

**Comment by maryjom** [comment]
May want to use "Replaced" rather than "Modified" because we only use this in 2 places where we replaced the notes with a significantly rewritten one.

**Comment by daniel-montalvo** [comment]
We have 'replaced' as well.

the classes we can use are:

- wcag2ict -- for a note that's added in the WCAG2ICT guidance 
- replaced -- for a note that has been "replaced"
- original -- for original WCAG notes
 - modified -- for notes that have been modified
 

As said [earlier in the PR](https://github.com/w3c/wcag2ict/pull/701#discussion_r2159275094), To me there is nuance around these two: "replaced" being completely rewritten whereas "modified" being slightly reworded.

**Comment by maryjom** [comment]
Decided to use "Replaced" as it was pretty extensively modified. Also added "Replaced" on the note for "accessibility supported" since we replaced 5 notes with one.
